### ISSUE TITLE
Issues with JPackage on Linux and Suggestions

### ISSUE THREAD

**Issue by lorenzo-gomez-windhover** [ControlSystemStudio/phoebus#1985]
Relates to #1983

So I ran the jpackage commands as instructed in the README, and it packages both rpm and deb. It installs it to /opt/product-name, which is great. The only problem is when I try to execute /opt/product-name/bin/exec_name, it gives me the following:

Failed to find JVM in "/opt/commander/lib/runtime" directory.

Now I can still run the product by going into /opt/commander/lib/app and running phoebus.sh, but I was hoping I could just execute the binary under bin.

I checked /opt/commander/lib/runtime/bin and it has all of the java executables.

Any idea what the problem might be?

Should I open up an issue about this?

The behavior is the same for rpm and deb.

One thing we could do(though no ideal because it assumes the JVM is installed on the user's machine) is make an `AppImage`. I know how to package those, if you're all interested I can always make a little README that explains how to package one.

Thanks
Lorenzo

**Comment by georgweiss** [comment]
Yes, we should maybe include a section on how to build a package without the Java runtime. I find bundling Java a significant benefit as it saves the user from installing the correct version, and it solves potential issues with any existing Java runtime that may not be compatible with the application. With IT management tools pushing updates, installed packages will be "overwritten", so users will not end up having multiple JVMs.
